data/samplesheets/190719_SN737_0468_ACDTBVACXX_spike.csv

```diff
@@ -0,0 +1,57 @@
+[Header],,,,,,,,,,,
+IEMFileVersion,5,,,,,,,,,,
+Date,07/17/19,,,,,,,,,,
+Workflow,GenerateFASTQ,,,,,,,,,,
+Application,HiSeq FASTQ Only,,,,,,,,,,
+Instrument Type,HiSeq 1500/2500,,,,,,,,,,
+Assay,Agilent_Sure_SelectXT,,,,,,,,,,
+Index Adapters,Agilent_Sure_SelectXT,,,,,,,,,,
+Description,PE-Amp-Lin-Block-Hyb-v 8.0,,,,,,,,,,
+Chemistry,default,,,,,,,,,,
+Operator,Daniel Scholtyssik / Katayoun Alemazkur /,,,,,,,,,,
+Investigator,Layal Yasin / Ute Fischer,,,,,,,,,,
+Backup_NAS_name,Ingerimm,,,,,,,,,,
+kind_of_run,2x101bp,,,,,,,,,,
+[Reads],,,,,,,,,,,
+101,,,,,,,,,,,
+101,,,,,,,,,,,
+,,,,,,,,,,,
+[Settings],,,,,,,,,,,
+ReverseComplement,0,,,,,,,,,,
+Adapter,AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,,,,,,,,,,
+AdapterRead2,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT,,,,,,,,,,
+,,,,,,,,,,,
+[Data],,,,,,,,,,,
+Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,Sample_Project,Description,spike_entity_id,spike_entity_role,spike_ignore_sample
+1,KB0268_c,,,,C01,AACGTGAT,Keimbahn,set1,KB0268,child,
```

Comment:
the entity name must be "patient" not "child". Spike fill otherwise fail to identify which sample belongs to the child vs. the parents!